controllers/controller.js

```diff
@@ -106,3 +107,13 @@ exports.deleteComment = (req, res, next) => {
     next(err);
   });
 };
+
+exports.getUsers = (req, res, next) => {
+  selectUsers()
+    .then((data) => {
+      res.status(200).send({ data });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
```

Comment:
Nice, but just like in the testing side data is not a good name for what is the array of users